xml_schema/isengard_contextitem.xsd

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE schema PUBLIC "-//W3C//DTD XMLSCHEMA 200102//EN" "http://www.w3.org/2001/XMLSchema.dtd">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="extension">
+    <xs:complexType>
+      <xs:all>
+        <xs:element name="item" type="itemType" minOccurs="1"/>
+      </xs:all>
+      <xs:attribute name="point" type="pointIdentifier" use="required"/>
+      <xs:attribute name="library" type="xs:string" use="required"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:complexType name="itemType">
+    <xs:all>
+      <xs:element name="label" type="xs:string" minOccurs="1"/>
+      <xs:element name="visible" type="xs:string" minOccurs="1"/>
+      <xs:element name="parent" type="parentIdentifier" minOccurs="0"/>
+    </xs:all>
+  </xs:complexType>
+  <xs:simpleType name="pointIdentifier">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="kodi.context.item"/>
```

Comment:
`kodi\.context\.item` for consistency with the other schemas (after rechi's comments)